src/renderer/components/dialog-bisect.tsx

```diff
@@ -118,39 +118,23 @@ export class BisectDialog extends React.Component<BisectDialogProps, BisectDialo
         <div className='bp3-dialog-body'>
           <Label>
             Earliest Version
-            <ElectronVersionSelect
-              filterable={true}
-              items={allVersions}
-              itemRenderer={renderItem}
-              itemPredicate={filterItem}
-              onItemSelect={this.onBeginSelect}
-              noResults={<MenuItem disabled={true} text='No results.' />}
-            >
-              <Button
-                text={startIndex ? `v${allVersions[startIndex].version}` : ``}
-                icon={startIndex ? getItemIcon(allVersions[startIndex]) : 'small-minus'}
-                fill={true}
+            <ButtonGroup fill={true}>
+              <VersionSelect
+                currentVersion={allVersions[startIndex]}
+                appState={this.props.appState}
+                onVersionSelect={this.onBeginSelect}
               />
-            </ElectronVersionSelect>
+            </ButtonGroup>
           </Label>
           <Label>
             Latest Version
-            <ElectronVersionSelect
-              filterable={true}
-              items={allVersions.slice(0, startIndex!)}
-              itemRenderer={renderItem}
-              itemPredicate={filterItem}
-              onItemSelect={this.onEndSelect}
-              noResults={<MenuItem disabled={true} text='No results.' />}
-              disabled={!startIndex}
-            >
-              <Button
-                text={endIndex ? `v${allVersions[endIndex].version}` : ``}
-                icon={endIndex ? getItemIcon(allVersions[endIndex]) : 'small-minus'}
-                fill={true}
-                disabled={!startIndex}
+            <ButtonGroup fill={true}>
```

Comment:
Now that there's no UI restriction preventing the Version Select from choosing a Latest Version that's earlier than the Earliest Version, could we add a ToolTip to the disabled button to explain the requirements to begin the Bisect?